runner/src/server/plugins/engine/page-controllers/ConfirmPageController.ts

```diff
@@ -21,7 +21,23 @@ export class ConfirmPageController extends SummaryPageController {
             const state = await adapterCacheService.getState(request);
             const fund_name = state["metadata"]["fund_name"];
             const round_name = state["metadata"]["round_name"];
-            return redirectTo(request, h, config.eligibilityResultUrl + "/" + fund_name + "/" + round_name);
+
+            let question_answer_short_code;
```

Comment:
My only very minor thought is that as this variable is used very specifically for the optional redirect feature it could be named something like `eligibleRoundRedirectAnswerShortCode` (although a bit of a mouthful!). Ideally it would follow idiomatic JavaScript and use camel case rather than snake case either way

Reply:
Great shout, let me update this